### DISCUSSION TITLE
Carousel only on mobile ?

### DISCUSSION THREAD

**Discussion by bckthomas** [fancyapps/ui#337] [Q&A]
Hi 
I would like to know if it's possible to enable carousel according to breakoint.
I can't find an option for this in the documentation.
Thanks

**Comment by fancyapps** [comment]
Hi,

What exactly you wish to customize depending on the screen size? If you want to change the number or visible slides, you set slide width using CSS.

**Reply by bckthomas** [reply]
I want to enable carousel on mobile only

**Reply by bckthomas** [reply]
More information : I can have between 1 and 6 elements in grid view on tablet and deskstop, and I want a slider on mobile
